sstable/options.go

```diff
@@ -15,8 +15,8 @@ import (
 )
 
 const (
-	// MaximumBlockSize is the maximum permissible size of a block.
-	MaximumBlockSize = rowblk.MaximumSize
+	// MaximumRestartOffset is the maximum permissible size of a block.
```

Comment:
nit: Maximum permissible value for a restart offset - i.e the maximum block size which allows adding a restart point.